src/class-bc-asaas.php

```diff
@@ -95,12 +102,32 @@ public function get( $endpoint, $data = array() ) {
 			return $response;
 		}
 
-		// if the response is not an image, let's json decode the body
+		// Get first response
 		$response = json_decode( wp_remote_retrieve_body( $response ) );
 
-		// When having data, return it already
-		// @todo consider pagination
+		$body_var = get_object_vars( $response );
+
+		// Return if we have only this data
 		if ( ! empty( $response->data ) ) {
+
+			$offset = 10;
+
+			// If has more values -> make loop
+			while ( $body_var['hasMore'] ) {
+
+				// Builds url sending offset var
+				$url = $this->build_url( $endpoint, array( 'offset' => $offset ) );
+
+				$page = json_decode( wp_remote_retrieve_body( wp_remote_get( esc_url_raw( $url ), $args ) ) );
```

Comment:
Avoid Huge lines like this, split it into two variables.
Also, the answer from `wp_remote_get` can be an `WP_Error`, so use `is_wp_error` to prevent Fatals.